package.json

```diff
@@ -1,25 +1,56 @@
 {
   "name": "electron-clear-data",
-  "version": "1.0.2",
-  "description": "Clears user data for your electron application",
+  "version": "1.0.3",
+  "description": "Securely clears user data for your Electron application.",
+  "homepage": "https://github.com/standardnotes/electron-clear-data#readme",
+  "repository": {
+    "type": "git",
+    "url": "https://github.com/standardnotes/electron-clear-data.git"
+  },
+  "bugs": {
+    "url": "https://github.com/standardnotes/electron-clear-data/issues",
+    "email": "security@standardnotes.org"
+  },
   "keywords": [
     "electron",
     "clear",
     "userdata",
     "delete",
     "wipe",
-    "application"
+    "application",
+    "leveldb"
   ],
   "license": "MIT",
   "author": "Johnny Almonte <johnny@standardnotes.org>",
-  "main": "main.js",
+  "main": "dist/main.js",
+  "typings": "dist/main.d.ts",
   "scripts": {
-    "test": "echo \"Error: no test specified\" && exit 1"
+    "test": "jest",
+    "coverage": "npm run test -- --coverage",
+    "watch": "tsc -w",
+    "build": "tsc",
+    "prepublish": "npm run build"
   },
   "devDependencies": {
-    "fs-extra": "^9.1.0"
+    "@jest-runner/electron": "^3.0.1",
+    "@types/fs-extra": "^9.0.10",
+    "@types/jest": "^26.0.22",
+    "@types/node": "^14.14.37",
+    "electron": "^9.4.0",
+    "fs-extra": "^9.1.0",
+    "jest": "^26.6.3",
+    "spectron": "^11.0.0",
+    "ts-jest": "^26.5.4",
+    "ts-node": "^9.1.1",
+    "typescript": "^4.2.4"
   },
   "peerDependencies": {
-    "electron": ">= 9.4.0"
+    "electron": "^9.4.0"
```

Comment:
Do we not want to be compatible with more recent electron releases? I know v9 is what our desktop app targets at the moment, but that is expected to change soon as well.

Reply:
That's a mistake. I've reverted it to `"electron": ">= 9.4.0"`